src/main/java/org/jabref/gui/maintable/MainTableColumnFactory.java

```diff
@@ -313,16 +322,29 @@ private void openUrlOrDoi(MouseEvent event, BibEntryTableViewModel entry, String
         event.consume();
     }
 
-    private TableColumn<BibEntryTableViewModel, String> createIconColumn(JabRefIcon icon, String field) {
+    private String createIdentifierTooltip(BibEntryTableViewModel entry, String content) {
+        Optional<String> field = entry.getEntry().getField(content);
+        if (field.isPresent()) {
+            if (FieldName.DOI.equals(content)) {
+                return Localization.lang("Open DOI web link (%0)", field.get());
```

Comment:
As DOI, Arxiv are not really translateable, I would just add them as parameter, e.g Open %0 Url  %1 (at least for arxiv)

Reply:
Done!